FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt

```diff
@@ -19,9 +19,12 @@ package com.android.samples.filemanager
 import android.content.Intent
 import android.net.Uri
 import android.os.Environment
+import android.util.Log
 import android.webkit.MimeTypeMap
 import androidx.appcompat.app.AppCompatActivity
-import java.io.File
```

Comment:
This import is required below (see line 32).

Reply:
Fixed in 25399f45767d7144815617ba012eb04bc9c0b449